agent/src/agent/cli/run_test_pipeline.py

```diff
@@ -0,0 +1,87 @@
+import os
+import time
+import click
+import sdc_client
+
+from jsonschema import ValidationError
+from agent import destination, pipeline, source
+from agent.cli.pipeline import check_prerequisites, get_info
```

Comment:
we prefer to import a module and then call its functions, like
```
from agent import cli
...
cli.pipeline.check_prerequisites()
```
it's more clear what the function does and may prevent circular dependencies in some cases